examples/integrations/cursor/README.md

```diff
@@ -17,6 +17,12 @@ Now, you can use Cursor as you would normally but with the `tensorzero::function
 This will send all traffic through your local TensorZero gateway.
 Take a look at the server running on `http://localhost:6901` to see what your requests look like!
 
+## Intercepting requests for cursor tab:
+
```

Comment:
this is OK for now but by the time this is ready for folks to use we should add some more expository writing or cut this